doc/source/examples/similarity_explanations_mnist.ipynb

```diff
@@ -0,0 +1,616 @@
+{
```

Comment:
<ol><li>Please add a header "Utils" so it's clear what this section is for</li><li>Please check the spelling and conventions again, non-exhaustive list:</li></ol><ul><li>Return -&gt; Returns</li><li>appendInt -&gt; append_int</li><li>"Converts integer into cardials" -&gt; "Convert an integer into an ordinal"</li><li>Oridinal -&gt; Ordinal</li><li>etc.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>